privacyIDEAADFSProvider/Adapter.cs

```diff
@@ -372,14 +350,19 @@ public void OnAuthenticationPipelineLoad(IAuthenticationMethodConfigData configD
                 // Only if triggerChallenge is disabled, sendEmptyPassword COULD be set
                 this.sendEmptyPassword = GetFromDict(configDict, "send_empty_pass", "0") == "1";
             }
-            this.realm = GetFromDict(configDict, "realm", "");
+
+            this.privacyIDEA.Realm = GetFromDict(configDict, "realm", "");
+            var realmmap = rr.GetRealmMapping();
+            Log("realmmapping: " + string.Join(" , ", realmmap));
+            this.privacyIDEA.RealmMap = realmmap;
```

Comment:
Shouldn't the `realmmap` actually be used somehwere?
To me it looks as if the "realm", that is sent to privacyidea is always the domain name that is fetched from the original login name.

Reply:
the realm is evaluated at Privacyidea.cs 358 (which is below, can't link lines on a PR....)

Reply:
Ah, ok. I was not aware that the instance attribute "RealmMap" does not have to be preceeded by anything.